packages/app/src/pages/api/feedback.ts

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2023 Frachtwerk GmbH, Leopoldstraße 7C, 76133 Karlsruhe.
+ *
+ * This file is part of Essencium Frontend.
+ *
+ * Essencium Frontend is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Essencium Frontend is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Essencium Frontend. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { FeedbackInput } from '@frachtwerk/essencium-types'
+import { NextApiRequest, NextApiResponse } from 'next'
+import nodemailer from 'nodemailer'
+
+export async function sendFeedbackEmail(
+  feedback: FeedbackInput,
+): Promise<void> {
+  const transporter = nodemailer.createTransport({
+    host: 'smtp.ethereal.email',
+    port: 587,
+    auth: {
+      user: 'eduardo23@ethereal.email',
+      pass: 'F4McAHa7kvNXbz7P5f',
+    },
```

Comment:
We need to outsource them into env-variables (host, port, user and pass). Those env vars are located inside `.env.local` then since they are sensitive. Here, this is no problem because of a dummy SMTP but for a real SMTP. Same is true for line `38` --> `to`.

Reply:
I tried it and it worked perfectly. Next.js supports the [`.env.local`](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#loading-environment-variables) out of the box. Let's talk about it in our weekly.

Reply:
I got it now :D